src/bundle/KiotaRequestAdapter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System.Net.Http;
+using Microsoft.Kiota.Abstractions;
+using Microsoft.Kiota.Abstractions.Authentication;
+using Microsoft.Kiota.Abstractions.Serialization;
+using Microsoft.Kiota.Http.HttpClientLibrary;
+using Microsoft.Kiota.Serialization.Form;
+using Microsoft.Kiota.Serialization.Json;
+using Microsoft.Kiota.Serialization.Multipart;
+using Microsoft.Kiota.Serialization.Text;
+
+namespace Microsoft.Kiota
```

Comment:
[Open for feedback]
This is exposed in the `Microsoft.Kiota` namespace

Reply:
The package name and the namespace name are generally aligned. Unless we have a good reason not to follow that convention, I'd suggest we do.

Reply:
This may also not work for other languages I believe. So, I'll update this in an upcoming update.